### PR TITLE
feat(stripe-dispute-orders): Add warning banner when fiscal host has disputed orders.

### PR DESCRIPTION
**Pull request by vaughanj10**
Resolve: https://github.com/opencollective/opencollective/issues/5983
Require: https://github.com/opencollective/opencollective-api/pull/8029

# Description

This PR adds a warning message on the home screen of the admin dashboard for a fiscal host when there are Orders that are disputed as potentially fraudulent by Stripe.

# Screenshots

image

### CODE REVIEW

File: components/host-dashboard/HostDashboardExpenses.js [hunk 1]
```diff
@@ -213,6 +213,17 @@ const HostDashboardExpenses = ({ hostSlug, isNewAdmin }) => {
...
+        <Box mb={4}>
+          <MessageBox type="warning" withIcon>
+            <FormattedMessage
+              id="host.disputes.warning"
+              defaultMessage="Fraud Warning: There are disputed charges that need review."
+            />{' '}
+            <Link href={`/${hostSlug}/admin/orders?status=DISPUTED`}>Review Disputes</Link>
+          </MessageBox>
+        </Box>
+      )}
```
Comment (kewitz) [inline components/host-dashboard/HostDashboardExpenses.js:226]: Suggestion: move this to `pages/admin-panel` and make it full-width above all the Grid component.

**Review by kewitz — APPROVED** [review]
LGTM.

**Review by znarf — DISMISSED** [review]
Seems to me that `hasDisputedOrders` looks at the Orders table and check if there is any `DISPUTED` orders. When a dispute is lost, I have not seen this `DISPUTED` status updated. I fear that the banner will be displayed forever once a dispute is lost?

Let me know if I'm missing something.

**Comment by vaughanj10** [comment]
@znarf  Ah, yes that is right. I'm going to resolve the Order status to `REJECTED` when the dispute is lost to make sure this problem doesn't happen.

**Pull request merged**